src/engine/Server.cpp

```diff
@@ -320,7 +320,11 @@ Awaitable<void> Server::process(
   } else if (auto cmd =
                  checkParameter("cmd", "dump-active-queries", accessTokenOk)) {
     logCommand(cmd, "dump active queries");
-    response = createJsonResponse(queryRegistry_.getActiveQueries(), request);
+    nlohmann::json json;
+    for (auto& [key, value] : queryRegistry_.getActiveQueries()) {
+      json[nlohmann::json(key)] = std::move(value);
+    }
```

Comment:
Let's see if we really can't fix this:)

Reply:
but super unimportant...

Reply:
I tried, it just doesn't work as expected